### PR TITLE
ROX-10018: Exclude service account oauth-apiserver-sa from "Kubeadmin Secret Accessed" policy

### PR DESCRIPTION
**Pull request by theencee**
## Description

This gets triggered by default on ARO as this is used by the apiserver for OAuth2

## Checklist
- [ ] Investigated and inspected CI test results
- [x] Unit test and regression tests added
- [x] Evaluated and added CHANGELOG entry if required
~[] Determined and documented upgrade steps~
- [ ] Documented user facing changes (create PR based on [stackrox/openshift-docs](https://github.com/stackrox/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

manual on ARO

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

**Comment by theencee** [comment]
Messed up rebase so I had to start from scratch on top of master. 

Also updated the test itself to _not fail_ if this policy is triggered if and only if all the violations are by the `kube:admin` user via `kubectl`. This is because this alert is legit. It's likely coming because in the CI pipeline we're accessing kubeadmin secret when uploading artifacts.

**Review by viswajithiii — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - ROX-10270: The `RenamePolicyCategory` and `DeletePolicyCategory` methods in the
 `v1/policycategories` endpoint have been deprecated, and will be removed in future releases. 
   - For questions about this change, please contact the Red Hat support team at support@redhat.com.
+- ROX-10018: The policy `OpenShift: Kubeadmin Secret Accessed` will not trigger for the default OpenShift `oauth-apiserver-sa` service account as that is an expected access pattern for the apiserver on OpenShift.
```
Comment (md2119) [inline CHANGELOG.md:37]: It reads like it won't whether or not an exclusion exists.

Comment (theencee) [reply CHANGELOG.md]: Updated wording

**Pull request merged**